game/script.rpy

```diff
@@ -1000,57 +1000,55 @@ if tutor:
+  "{b}Minos{/b}: You covet my bull? The mighty bull of Poseidon?"
 
-  "{b}Minos{/b}: Love!? So light a thing you think to say! There must be some explanation for this, some supernatural cause. Some curse has come upon your mind."
+  "{b}Pasiphaë{/b}: King you are, but queen am I, what great {i}blessings{/i} have been {i}showered{/i} upon you. What {i}great fortune{/i}. Never have I received anything I have not grasped with my own two hands."
 
-  "{b}Pasiphaë{/b}: Should I want but a single thing and you assume I am cursed? On what grounds? What law? What word? Am I so strange that you would not allow me this one thing?"
+  "{b}Chorus{/b}: A house quiet sits in balance with the gods. Fortunate are they who can remain content. But when that quiet is broken, that house is shaken forever."
 
-  "{b}Chorus{/b}: What preparation does he have to draw this water? What strength has he built in his life? Some fail, some falter, and let the rope fall."
+  "{b}Minos{/b}: What fearsome jealousy! It twists the woman before me. So sudden it rises. How has this evil taken hold?"
 
-  "{b}Minos{/b}: I do not know you, I do not see. It is not one, but two wild creatures before me. Surely this is not you, it never is or was or will be you. Nothing of this is like the woman I wed."
+  "{b}Pasiphaë{/b}: Sudden? Was it not I who cursed your seed to serpents? When a woman dresses her desires as lover's envy, her husband sees nothing. Yet when she wishes power, she is now jealous?"
 
-  "{b}Pasiphaë{/b}: And what would you know what I am like? Maybe I am who I always was. Who I wasn't letting myself become. You never saw me. You saw who you wanted me to be."
+  "{b}Chorus{/b}: Love is the lightless cave man walks down, blind to every stone in his path. Cut and bruised he can't see who assaults him, deeper in he goes, following only the promise of a soft-faced girl."
 
-  "{b}Minos{/b}: Be you god, be you mortal, be you sorceress in between. The one I wed is human. The one I wed is loyal to me."
+  "{b}Pasiphaë{/b}: Give me your bull, or I shall take it. You cannot keep a god's gift for yourself. It is I who have just as much right to it as you."
 
-  "{b}Pasiphaë{/b}: How blind you are! How deep this ache! This human world is no place for me. The one I wed, the one I see, slays the love within me."
+  "{b}Minos{/b}: I do not know you, I do not see. It is not one, but two wild creatures before me. Be you god, be you mortal, be you sorceress in between. Nothing of this is like the woman I wed. The one I wed is content with me."
 
-  "This certainly does not sound like the immortal sorceress Pasiphaë I'd read about."
+  "{b}Pasiphaë{/b}: And what would you know what I am like? Maybe I am who I always was. You never saw me. You saw who you wanted me to be. The one I wed, the one I see, slays the love within me."
```

Comment:
I think this could work with just the last line, "The one I wed...slays the love within me"